Examples/create-app.js

```diff
@@ -61,6 +61,17 @@ let iosStagingDeploymentKey = null;
 createCodePushApp(appNameAndroid, 'Android');
 createCodePushApp(appNameIOS, 'iOS');
 
+if (!isReactNativeVersionLowerThan(60) && process.platform === "darwin") {
```

Comment:
Can we please move this code close to start, so that install necessary dependencies before creating apps both in App Center and disk?

Reply:
No we can't. I tried to move it in linkCodePush() method after checking that platform is MacOS but script failed in generatePlainReactNativeApp() method.

Reply:
Sorry, am I missing something? From my understanding it should be possible to:
1. move line 47 to line 43
2. place this newly added check after line 43.

This way we should be able to avoid apps creation when not all the dependencies are set up

Reply:
I understand. Thank you for review.